### ISSUE TITLE
CORS request not working to production server.

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/dockless#32]
I'm seeing this error when in I try to pull data from a staged client app on Netlify.

`Access to fetch at 'https://dockless-data.austintexas.io/v1/trips?xy=-97.74394821167274,30.271997902824353&flow=origin&mode=all' from origin 'https://deploy-preview-31--dockless.netlify.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

After some Google-fu, [here](https://stackoverflow.com/questions/36878255/allow-access-control-allow-origin-header-using-html5-fetch-api) and [here](https://stackoverflow.com/questions/10627703/d3-json-d3-xhr-and-cross-domain-problems), it seems that we need to make the request to the backend with a header, and/or pass options into the request. 

Maybe the error is misleading and the server needs to be configured to accept CORS requests, but I thought we had already done that.

Possible solution: replace d3-fetch with another ajax library

**Comment by mateoclarke** [comment]
Going to wait on a redeploy of the most up-to-date dockless-api Python code on the server-side before changing anything about the front-end app.

**Comment by mateoclarke** [comment]
This is now resolved with the latest push of backend code.
